scipy/stats/_qmc.py

```diff
@@ -914,13 +910,38 @@ class LatinHypercube(QMCEngine):
 
         .. versionadded:: 1.8.0
 
+    orthogonal : bool, optional
+        Orthogonal array based LHS of strength 2 [7]_, [8]_.
+        When used, only ``n=p**2`` points can be sampled, with ``p`` a prime
+        number. It also constrains ``d <= p + 1``.
+        Default is False.
+
+        .. versionadded:: 1.8.0
+
     seed : {None, int, `numpy.random.Generator`}, optional
         If `seed` is None the `numpy.random.Generator` singleton is used.
         If `seed` is an int, a new ``Generator`` instance is used,
         seeded with `seed`.
         If `seed` is already a ``Generator`` instance then that instance is
         used.
 
+    Notes
+    -----
+
+    LHS is extremely effective on integrands that are nearly additive [2]_.
+    LHS on :math:`n` points never has more variance than plain MC on
+    :math:`n-1` points [3]_. There is a central limit theorem for LHS [4]_,
+    but not necessarily for optimized LHS.
+
+    LHS is an orthogonal array of strength 1.
+    :math:`A` is called an orthogonal array of strength :math:`t` if in each
+    n-row-by-t-column submatrix of :math:`A`, all :math:`q^t` possible
+    distinct rows occur the same number of times.
+
+    Strength 1 (LHS) brings an advantage over strength 0 (MC) and strength 2
```

Comment:
```suggestion
    Strength 1 (LHS) brings an advantage over strength 0 (MC) and strength 2
```
By "LHS", this means `orthogonal=False` LHS?
Strength 2 is (`orthogonal=True` LHS)?

Reply:
Yes

Reply:
: / but resolving. We can discuss this [above](https://github.com/scipy/scipy/pull/14546#pullrequestreview-728203108).